quantum/process_keycode/process_autocorrect.c

```diff
@@ -67,6 +67,23 @@ void autocorrect_toggle(void) {
  * @return false Stop processing and escape from autocorrect.
  */
 __attribute__((weak)) bool process_autocorrect_user(uint16_t *keycode, keyrecord_t *record, uint8_t *typo_buffer_size, uint8_t *mods) {
+    return preprocess_autocorrect(keycode, record, typo_buffer_size, mods);
+}
+
+/**
+ * @brief fallback handler for determining if autocorrect should process this keypress
+ *        can be used by user callback to get the basic keycode being "wrapped"
+ *
+ * NOTE: These values may have been edited by user callback before getting here
+ *
+ * @param keycode Keycode registered by matrix press, per keymap
+ * @param record keyrecord_t structure
+ * @param typo_buffer_size passed along to allow resetting of autocorrect buffer
+ * @param mods allow processing of mod status
+ * @return true Allow autocorection
+ * @return false Stop processing and escape from autocorrect.
+ */
+bool preprocess_autocorrect(uint16_t *keycode, keyrecord_t *record, uint8_t *typo_buffer_size, uint8_t *mods) {
```

Comment:
preprocess feels ... incorrect here, given that it is used elsewhere. 

`process_autocorrect_handler` might be better?  But naming... it's difficult.

Reply:
Yeah, i was actually thinking to call this first and then the user one, but since that may prevent users from doing other overriding i changed the logic but forgto about the name....